common/language.c

```diff
@@ -331,6 +331,28 @@ const char* const g_strings[StrId_Max][16] =
         STR_TW("应用"),
     },
 
+    [StrId_Actions_Star] =
+    {
+        STR_EN("Star"),
+        STR_FR("Star"),
```

Comment:
Remove dups, STR_EN will be used when an entry isn't set (currently the used language is hard-coded to EN anyway).

Reply:
Done